### DISCUSSION TITLE
Dublicate table. Why so crazy hard.

### DISCUSSION THREAD

**Discussion by andreysneg** [dbeaver/dbeaver#20547] [Q&A]
In heidisql dublicate table take 2 clicks and 5 seconds.

In DBeaver is so hard what make me crazy. And this is PRO software ?

After many minutes and google and documenation read result was - erros. Dbveaer cant create dublicate. In heidisql a make copy of this table in 5 seconds and 2 clicks.

Picture with error. Whats wrong ?
https://bit.ly/46GfEqN

**Comment by LonwoLonwo** [comment]
Hello @andreysneg 

May be Data Migration page will be helpful: https://github.com/dbeaver/dbeaver/wiki/Data-migration

**Comment by andreysneg** [comment]
I did it. but dbeaver show - SQL error
https://bit.ly/46GfEqN

**Reply by LonwoLonwo** [reply]
The image is not available.
Could you please provide the log files? You can find instructions here: https://dbeaver.com/docs/wiki/Log-files/. Thank you.

**Comment by andreysneg** [comment]
I found problem. i must select database too.  Database by default must automaticaly select some as copy table.  it's intuitive.

You can close the topic. Thank you.
